### PR TITLE
Nikolay/review/clean_up

### PR DESCRIPTION
**Pull request by kipod**
review and fix some isues

### CODE REVIEW

File: app/templates/auth/login.html [hunk 1]
```diff
@@ -1,25 +1,78 @@
...
+            >
+              Sign In
+            </button>
+          </div>
+        </form>
+      </div>
     </div>
   </div>
-</div>
-{% endblock %}
```
Comment (DenisBurimov) [inline app/templates/auth/login.html:25]: It would be great if even in html it will be empty row

File: app/templates/auth/register.html [hunk 2]
```diff
@@ -1,45 +1,94 @@
...
+              Already have an account?
+              <a href="{{ url_for('auth.login') }}">Login</a>.
+            </p>
+          </div>
+        </form>
+      </div>
     </div>
   </div>
-</div>
-{% endblock %}
```
Comment (DenisBurimov) [inline app/templates/auth/register.html:45]: It would be great if even in html it will be empty row

File: app/templates/base.html [hunk 3]
```diff
@@ -1,127 +1,60 @@
...
-    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-  <link href="{{ url_for('static', filename='css/main.css') }}" rel="stylesheet" media="screen">
-  {% block links %}{% endblock %}
-</head>
+  <body class="relative h-screen bg-blueGray-50">
+    <!-- Header -->
+    {% include 'header.html' %}
+    <!-- Flash Messages -->
+    {% with messages = get_flashed_messages(with_categories=true) %} {% if
+    messages %} {% for category, message in messages %}
```
Comment (DenisBurimov) [inline app/templates/base.html]: There are cases, when if you move jinja logic to another row, it doesn't work. Not a best option

File: app/templates/base.html [hunk 4]
```diff
@@ -1,127 +1,60 @@
...
-  <script src="{{ url_for('static', filename='js/main.js') }}" type="text/javascript"></script>
-  {% block scripts %}{% endblock %}
-</body>
+    {% endfor %} {% endif %} {% endwith %}
+    <!-- Main Content -->
+    {% block content %}{% endblock %}
+    <!-- Footer-->
+    {% include 'footer.html' %}
 
-</html>
```
Comment (DenisBurimov) [inline app/templates/base.html:127]: It would be great if even in html it will be empty row

File: config.py [hunk 5]
```diff
@@ -14,6 +14,15 @@ class BaseConfig(object):
...
     WTF_CSRF_ENABLED = False
 
+    # Mail config
+    MAIL_SERVER = os.getenv("MAIL_SERVER", "mail.simple2b.com")
+    MAIL_PORT = int(os.getenv("MAIL_PORT", "465"))
+    MAIL_USE_TLS = False
+    MAIL_USE_SSL = True
+    MAIL_USERNAME = os.getenv("MAIL_USERNAME", "admin@simple2b.com")
+    MAIL_PASSWORD = os.getenv("MAIL_PASSWORD", "!need_set_pass!")
+    MAIL_DEFAULT_SENDER = os.getenv("MAIL_DEFAULT_SENDER", "admin@simple2b.com")
```
Comment (DenisBurimov) [inline config.py]: I think, we can't afford to put credentials under the source control

File: tests/test_auth.py [hunk 6]
```diff
@@ -30,17 +17,46 @@ def test_auth_pages(client):
...
-            username="sam",
-            email="sam@test.com",
-            password="password",
-            password_confirmation="password",
-        ),
-        follow_redirects=True,
-    )
-    assert b"Registration successful. You are logged in." in response.data
+    # TEST_EMAIL = "sam@test.com"
+    TEST_EMAIL = "saintkos117@gmail.com"
```
Comment (DenisBurimov) [inline tests/test_auth.py]: Keep your personal data carefully

File: tests/test_auth.py [hunk 7]
```diff
@@ -30,17 +17,46 @@ def test_auth_pages(client):
...
+        assert "Confirm registration" in letter.html
+        assert user.unique_id in letter.html
+        html: str = letter.html
+        pattern = r"https?:\/\/[\w\d\.-]+\/activated\/[\w\d-]{36}"
+        urls = re.findall(pattern, html)
+        assert len(urls) == 1
+        url = urls[0]
+        response = client.get(url)
+        assert response.status_code == 302
+        response.location == url_for("main.index")
```
Comment (DenisBurimov) [inline tests/test_auth.py:55]: Maybe the better idea could be use follow_redirect attribute

File: pyproject.toml [hunk 8]
```diff
@@ -5,22 +5,16 @@ description = ""
...
-WTForms = "^3.0.1"
-itsdangerous = "^2.1.2"
-Jinja2 = "^3.1.1"
-MarkupSafe = "^2.1.1"
-python-dotenv = "^0.20.0"
-SQLAlchemy = "^1.4.32"
-Werkzeug = "2.0.0"
-psycopg2-binary = "^2.9.3"
-Flask-Migrate = "^3.1.0"
+python = "^3.11"
```
Comment (DenisBurimov) [inline pyproject.toml:8]: I am a supporter of strict versions in pyproject.toml

File: tailwind.config.js [hunk 9]
```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./app/templates/*.html", "./app/templates/auth/*.html"],
```
Comment (DenisBurimov) [inline tailwind.config.js]: Also could be convenient to use something like "./templates/**/*.{html}"

**Review by DenisBurimov — APPROVED** [review]
Well done

File: pyproject.toml (hunk 8 above)
Comment (kipod) [reply pyproject.toml:8]: ^3.11 means also 3.11.2, 3.11.3 etc

File: app/templates/base.html (hunk 3 above)
Comment (kipod) [reply app/templates/base.html]: added prettier-ignore

File: config.py (hunk 5 above)
Comment (kipod) [reply config.py]: ok

File: tests/test_auth.py (hunk 6 above)
Comment (kipod) [reply tests/test_auth.py]: fixed

File: tests/test_auth.py (hunk 7 above)
Comment (kipod) [reply tests/test_auth.py:55]: In this case we test correct HTTP redirection logic

File: tailwind.config.js (hunk 9 above)
Comment (kipod) [reply tailwind.config.js]: thank you. fixed

**Pull request closed without merging**